src/planner.cc

```diff
@@ -934,6 +938,23 @@ void Planner::Impl::SetExternalPosition(GCodeParserAxis axis, float pos) {
   }
 }
 
+bool Planner::Impl::SetLookAhead(int *size) {
+  if (size == nullptr) {
+    return false;
+  }
+  const int value = *size;
+  if (value == 0) {
+    *size = lookahead_size_;
+    return false;
+  }
+  if (value < 0 || value > PLANNING_BUFFER_CAPACITY) {
```

Comment:
This allows to set the planning buffer to zero. Would that work, or do we need at least one slot ?